apache/superset
# feat: Removes the Multiple Line Charts

### SUMMARY
This PR removes the Multiple Line Charts.

- Removes the legacy chart code and unregisters the plugin.
- Removes old plugin references throughout the codebase
- Updates unit tests
- Updates E2E tests
- Updates the example dashboards
- Updates translation files

The Multiple Line Charts description says that it's deprecated and people should use the Mixed chart instead. The problem is that Multiple Line Charts allows you to select 2 charts which may reference different datasets while Mixed Chart only allows 1 dataset. This means that to compare values from different datasets using a Mixed chart you would need to create another dataset combining the source datasets. Given that this chart is marked as deprecated for a long time, a migration would required significant effort, and its adoption is pretty low, we decided to remove it. If someone still has this chart and tries to access it, the following screen will be displayed:

<img width="1776" alt="Screenshot 2023-05-04 at 09 58 54" src="https://user-images.githubusercontent.com/70410625/236212527-22d4e736-7472-4e55-ace1-49acd4ea543b.png">

### AFTER SCREENSHOT
<img width="1079" alt="Screenshot 2023-05-04 at 09 46 19" src="https://user-images.githubusercontent.com/70410625/236210322-35219cc6-4d3e-4d88-b020-e8f623be01c4.png">

### TESTING INSTRUCTIONS
1 - Make sure all Multiple Line Charts were removed
2 - Make sure Multiple Line Charts is not available anymore in the viz picker

### ADDITIONAL INFORMATION
